src/index.js

```diff
@@ -9,6 +9,7 @@ import { useExternal } from './deprecated';
 import { getLogger } from './logger';
 import { PushWorker } from './Push/PushWorker';
 import { ParseServerOptions } from './Options';
+import * as Migrations  from './Migrations';
 import { ParseGraphQLServer } from './GraphQL/ParseGraphQLServer';
```

Comment:
not sure to understant the purpose of migrations, it seems to be some duplicated code of defined schema ?

Reply:
This is your `buildSchema.js` I just moved it under ParseServer import.


https://github.com/Moumouls/next-atomic-gql-server/blob/master/src/schema/buildSchemas.ts#L1

Reply:
Yeah, yeah, I did not read the hundreds of comments on your original PR and just followed your `working example`in the PR description 😅 I'll fix this today!